src/main/java/com/yieldlab/gdpr/util/GdprSwitch.java

```diff
@@ -0,0 +1,34 @@
+package com.yieldlab.gdpr.util;
+
+import static java.time.format.DateTimeFormatter.ISO_OFFSET_DATE_TIME;
+
+import java.time.Instant;
+import java.util.Optional;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.yieldlab.gdpr.GdprConsent;
+
+public class GdprSwitch {
```

Comment:
Not sure if all the methods in this class are really helpful. The only thing in which we're interested is the switch time. `isVendorAllowed` seems to be very special, after all.

Reply:
And if we want to contribute back, `isOn` should not be here but in the app IMO, as this library should only be about the consent string.

Reply:
I have removed this file from the project and inlined it in adserver.